src/legacy/core_plugins/kibana/public/dashboard/np_ready/dashboard_app_controller.tsx

```diff
@@ -254,6 +258,41 @@ export class DashboardAppController {
       };
     };
 
+    $scope.showFilterBar = () =>
```

Comment:
Do we need to put this on the scope? Seems like it not used anymore from within angular